### DISCUSSION TITLE
DST?

### DISCUSSION THREAD

**Discussion by Lord-Volkner** [Stellarium/stellarium#3739] [Q&A] [answered]
Stellarium 24.1 is not adjusting for DST. I have the location set correctly and the "Enable daylight saving time" box checked, but still it doesn't adjust for DST.

**Comment by gzotti** [comment]
And what is your time zone?

**Reply by Lord-Volkner** [reply]
Central Standard Time ... Well, technically it's Central Daylight Time right now.

**Reply by gzotti** [reply]
What is the name of the timezone that you have set in Stellarium's location panel?

**Reply by Lord-Volkner** [reply]
UTC-06:00 at the moment, but I've tried various different ones.

I tried America/Mexico City, which is nowhere near me, but it's in the same time zone, still reads an hour behind.

I also tried America/Chicago (also nowhere near me) which sets the time right, but changes it to UTC -05:00 which is not correct. Maybe that's fine, but I don't really know how the time thing works in Stellarium.

**Reply by gzotti** [reply, marked as answer]
I hope you have, being unsure about how timezones work in Stellarium,  already found section 4.2.1 in the User Guide. Here is some redundant and additional information.

UTC-6 etc. are settings that are just that. There are no daylight saving rules defined. 

Timezones with DST rules look like Continent/City. You take that city that defines your regional timezone rules. Probably the capital of your federal state if you are in the USA, or your capital city in most other countries of the world.

According to https://www.timeanddate.com/time/zones/cst, CST is UT-6. CDT=UT-5. America/Chicago has CST or CDT when applicable, with dates regulated by civil authorities. You can decide to remain in CST by disabling DST.

Mexico has abolished DST, therefore keeps UT-6. (https://www.timeanddate.com/time/change/mexico/mexico-city) You will not switch to CDT in 2024 when using timezone America/Mexico_City.

**Reply by Lord-Volkner** [reply]
Thank you.
